Extra_task_1/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="shortcut icon" href="favicon.ico" type="image/x-icon">
+    <link rel="stylesheet" href="style.css">
+    <title>Extra task 1</title>
+</head>
+<body>
+    <div class="background"></div>
+    <main class="modal">
+        <section class="modal__content">
+            <section class="modal__header">
+                <h3 class="modal__title">
+                    Modal title
+                </h3>
+                <a href="#" class="modal__close">
+                    <svg class="modal__icon" width="16" height="16" viewBox="0 0 16 16" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
```

Comment:
you can use HTML special characters for close symbol (&#10006;) https://html5book.ru/specsimvoly-html/